src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/use-incident-intelligence.mdx

```diff
@@ -33,15 +33,15 @@ In more detail:
 * **Incidents** are groups of events that describe the "symptoms" of your system over time. These symptoms are detected by your monitoring tools, which evaluate your data streams and events.
 * **Issues** are groups of incidents that describe the underlying problem of your symptoms. When a new incident is created, Incident Intelligence opens an issue and evaluates other open issues for correlations.
 
-## Issue summary [#issue-summary]
+## Issue details [#issue details]
```

Comment:
```suggestion
## Issue details [#issue-summary]
```

We should keep the anchor tags the same, unless it feels really important to change them. People don't page that much attention to URLs generally.